_pages/decide/comparative-analysis.md

```diff
@@ -31,3 +31,6 @@ To identify competitors’ solutions that excel, are lacking, or are missing cri
 ## Applied in government research
 
 No PRA implications. No information is collected from members of the public.
+
+## Examples from 18F
+ - [Draft Web Design Standards Comparative Analysis] (https://github.com/18F/web-design-standards/wiki/Comparative-Analysis)
```

Comment:
This should be `Draft U.S. Web Design Standards Comparative Analysis` even though the original omits the `U.S.`